src/App.js

```diff
@@ -13,8 +14,9 @@ const App = () => (
     <Routes>
       <Route path="/" element={<Layout />}>
         <Route index element={<Home />} />
+        <Route path="/lawyers/:id" element={<Lawyer />} />
+        <Route path="/reservations" element={<Reservations />} />
```

Comment:
Currently, the routes for the `Lawyer `and `Reservations` are children of the route with path "/", so for all of their child routes, prepend the "/" in the path prop is not necessary.
```suggestion
        <Route path="lawyers/:id" element={<Lawyer />} />
        <Route path="reservations" element={<Reservations />} />
```